students/jarosz_natalia/lesson_09/extending_the_multiclipboard.py

```diff
@@ -0,0 +1,37 @@
+#! python3
+# mcb.pyw - Saves and loads pieces of text to the clipboard.
+# Usage: py.exe mcb.pyw save <keyword> - Saves clipboard to keyword.
+#        py.exe mcb.pyw <keyword> - Loads keyword to clipboard.
+#        py.exe mcb.pyw list - Loads all keywords to clipboard.
+
+import shelve
+import pyperclip
+import sys
+
+
+def main():
+    mcb_shelf = shelve.open('mcb')
+
+    # TODO: Save clipboard content.
+    if len(sys.argv) == 3 and sys.argv[1].lower() == 'save':
+            mcb_shelf[sys.argv[2]] = pyperclip.paste()
+    elif len(sys.argv) == 2:
+
+        # TODO: List keywords and load content.
+        if sys.argv[1].lower() == 'list':
+            pyperclip.copy(str(list(mcb_shelf.keys())))
+        elif sys.argv[1] in mcb_shelf:
+            pyperclip.copy(mcb_shelf[sys.argv[1]])
+
+    # Extend the multiclipboard program in this so that it has a delete
+    # #<keyword> command line argument that will delete
+    # a keyword from the shelf.
+    elif len(sys.argv) == 3 and sys.argv[1].lower() == 'delete':
+        if sys.argv[2] in mcb_shelf:
+            mcb_shelf.pop(sys.argv[2])
+
```

Comment:
Missing of the second part of this task:
Then add a delete command line argument that will delete all keywords.
```
elif len(sys.argv) == 2 and sys.argv[1].lower() == 'delete':
    for element in list(mcbShelf.keys()):
        del mcbShelf[element]
```